content/300-guides/050-database/25-providers/provider-planetscale

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Prisma with PlanetScale'
+metaTitle: 'Prisma with PlanetScale'
+metaDescription: 'Guide on using Prisma with PlanetScale'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma and PlanetScale together provide a development arena that optimizes rapid, type-safe development of data access applications, using Prisma's ORM and PlanetScale's highly scalable MySQL-based platform.
+
+</TopBlock>
+
+## About using Prisma with PlanetScale
+
+This document discusses the concepts behind using Prisma and PlanetScale, explains some important concepts and considerations on the commonalities and differences between PLanetScale and other dtabase providers, and leads you through the process for configuring your application to implement PlanetScale.
+
+Two main benfits of using PlanetScale with Prisma are:
+
+- enterprise scalability
+    - <use Jan's content>
+    
+- support non-blocking schema changes
+    - <use Jan's content>
+    
+Watch our video about [Best Practices](https://www.youtube.com/watch?v=iaHt5_hg44c) when using Prisma with PlanetScale.
+    
+### Commonalities with other database proviers
+
+<use Jan's content>
+
+### Differences to consider
+
+<use Jan's content, cover FK since PS doesn't enforce FK, etc>
+
+### Additional considerations
+
+<add note about `db push` instead of `prisma migrate`>
```

Comment:
I think that is the _real_ content of the guide and what people will want to read to learn how to use Prisma with PlanetScale